\documentclass{article}
\usepackage{natbib}
%\usepackage{lipsum}
\usepackage{natbib}
\usepackage{bibentry}
\usepackage{hyperref}
\usepackage[english]{babel}
\usepackage[utf8]{inputenc}
%\usepackage[T1]{fontenc}
\begin{document}
\section*{Nicola Prezza: Scientific Curriculum}
\subsection*{Personal data}
%Født oktober 1984 og opvokset i Espergærde. Mor til Samuel på 2 år.
%Jeg kan træffes på :\\
\begin{tabular}{p{2cm} p{10cm}}
Name & Nicola Prezza\\
Address & University of Pisa, Largo Bruno Pontecorvo, 3 - 56127 - Pisa (Italy)\\
Email & nicola.prezza@gmail.com\\
Webpage & \url{https://nicolaprezza.github.io/}\\
%Telephone n. & 60943251\\
OrcID & \url{http://orcid.org/0000-0003-3553-4953}\\
H-index & 6, according to Google Scholar.
\end{tabular}

\subsection*{Education}
\begin{tabular}{p{2cm} p{10cm}}
1/1/2014--31/12/2016 & \textbf{Ph.D, Computer Science, Mathematics, and Physics, University of Udine.}\\
& Ph.D. thesis: ``Compressed Computation for Text Indexing''.\\
& Supervisor: Alberto Policriti.\\
26/9/2011--18/10/2013 & \textbf{Master’s degree in Computer Science, University of Udine.}\\
&Final grade: 110/110 cum laude. GPA: 3.90.\\
27/8/2012--30/1/2013 & \textbf{Erasmus Exchange student, University of Southern Denmark, Odense.}\\
25/9/2008-14/12/­2011 & \textbf{Bachelor’s degree in Computer Science, University of Udine.}\\
& Final grade: 110/110 cum laude. GPA: 3.73.
\end{tabular}

\subsection*{Employments}
\begin{tabular}{p{2cm} p{10cm}}
	1/3/2018--&\textbf{PostDoc researcher, dept. of Computer Science, University of Pisa}\\
	1/1/2017--28/3/2018&\textbf{PostDoc researcher, AlgoLoG, DTU Compute, DTU, Denmark.}\\
	1/1/2014--31/12/2016 & \textbf{Ph.D.-student, Computer Science, University of Udine.}\\
\end{tabular}

\subsection*{Teaching Experience}

\begin{tabular}{p{2cm} p{10cm}}
	15/9/2017--8/12/2017 & Ph.D course: \emph{Compact data structures}. 5 ECTS. Technical University of Denmark, Kgs. Lyngby, Denmark. \url{www2.compute.dtu.dk/courses/02951/}\\
	24/7/2017--28/7/2017 & \emph{Aligning DNA sequences on compressed collections of genomes} (Summer school). The CODATA-RDA Research Data Science Applied workshops on Extreme sources of data, Bioinformatics and IoT/Big-Data Analytics, ICTP, Trieste. Teaching hours: 6.\\
	Fall 2015 & \emph{Laboratorio di Architettura degli
	Elaboratori}. 3 ECTS. University of Udine.\\
	%Fall 2015 & \emph{Compressed indexing}. 3-lectures module within the MD course \emph{Algorithmica}, University of Udine.\\
	%Fall 2015 & \emph{Compression and indexing}. 1-lecture module within the MD course \emph{Teoria dell'informazione e crittografia}, University of Udine.\\	
	Fall 2014 & \emph{Laboratorio di Architettura degli
	Elaboratori}. 3 ECTS. University of Udine.\\
	%Fall 2014 & \emph{Self-indexing}. 3-lectures module within the MD course \emph{Algorithmica}, University of Udine.\\	
	23/10/2014 & \emph{Sequence Analysis for Epigenomics}. 2nd Bioinformatics Introductory Course, Polo d’Innovazione Genomica, Genetica e Biologia, Perugia. Teaching hours: 3.\\
\end{tabular}

\subsection*{PhD schools/visits abroad:}

\begin{tabular}{p{2cm} p{10cm}}
	9/8/2016--12/8/2016 & Summer School on Bioinformatics Data Structures, Helsinki.\\
	27/9/2015--1/10/2015 & School on Cancer, Evolution and Complexity, Como.\\
	1/6/2016--1/8/2016 & Visiting PhD student, Courant institute, NYU, New York.\\
	28/9/2014--2/10/2014 & School on Cancer, Systems and Complexity, Como.\\
	24/3/2014--28/3/2014 & Visiting PhD student, SciLifeLab, Stockholm.
\end{tabular}

\subsection*{Awards:}
\begin{tabular}{p{2cm} p{10cm}}
	20/06/2016 & Best talk award, MatBio 2016, King's college London.\\
\end{tabular}

\subsection*{Program Committees}
I am on the PC of the 29th Annual Symposium on Combinatorial Pattern Matching (CPM 2018) and I will chair the 13th Workshop on Compression, Text and Algorithms (WCTA 2018).


\subsection*{Reviewer activities}

I reviewed more than 35 publications for international conferences and journals. Conferences include: CPM, SPIRE, LATA, DCC, SEA. Journals include: Algorithmica, Information Systems, Theoretical Computer Science, Journal of Discrete Algorithms, Algorithms for Molecular Biology.


%\subsection*{Overview of scientific achievements}
%
%My scientific curriculum is a good match with the offered research position since I have strong competencies in theoretical computer science, bioinformatics, and programming. I have excellent connections with international researchers in my field; this has already led to several joint papers (read section \emph{Publications}). Most of these collaborations are ongoing and will lead to more joint papers in the near future.  
%I moreover gave talks at more than 25 international conferences and workshops (read section \emph{Talks at international venues}).
%My skills are confirmed by several important results that I have obtained with my research and that are summarized below.
%
%\paragraph*{Theoretical Computer Science}
%
%Within theoretical computer science, my main interests span (but are not limited to) stringology and algorithms and data structures for compression and indexing. I am author of the first in-place algorithm for the sparse suffix sorting problem~\cite{prezzaSparse}, which I will present at SODA 2018. In addition, in a joint work with the Diego Portales University and the University of Chile, I contributed at solving a long-standing problem related to indexes based on the Burrows-Wheeler transform~\cite{prezzaOptimal}. In this paper, we showed that a compressed suffix array sampling can be used to locate patterns in log-logarithmic time on an FM-index. The new strategy is orders of magnitude faster than existing ones and we expect it will have a great impact in several fields, including bioinformatics. Also this work will be presented by me at SODA 2018. 
%
%In my PhD thesis I solved several open problems related to the construction of compressed text indexes in compressed space. These results led to several publications at international conferences~\cite{policriti2015fast2, policriti2017lz77, belazzougui2017flexible, policriti2016computing, belazzougui2015composite, policriti2015average} and one in the journal Algorithmica~\cite{Policriti2017}.
%
%Finally, in my most recent work~\cite{kempa2017roots} (currently under review) I introduce a new class of combinatorial objects --- \emph{string attractors} --- that (i) provide a new powerful way of proving combinatorial properties of strings --- e.g. I use them to show the first relation between the Lempel-Ziv parsing and the Burrows-Wheeler transform ---, (ii) show for the first time that all dictionary compressors are approximations to the same problem (i.e. computing the smallest attractor), and (iii) represent a powerful tool in compressed computation --- I use them to close the random access problem on most dictionary compression schemes. This is a very promising line of research that I believe will lead to many more new surprising results in the fields of stringology and compressed computation. This is a joint work with a researcher from the university of Helsinki.
%
%\paragraph{Bioinformatics}
%
%I have strong competencies in DNA read alignment and DNA methylation analysis. I am among the authors of the short-reads aligner \url{erne.sourceforge.net} (I developed all the components related to the underlying compressed index). This work led to the publications
%\cite{prezza2012erne,policriti2015fast1}. Later, I extended our aligner to deal with bisulfite-treated DNA reads and developed a methylation caller. This work led to the Journal publication~\cite{prezza2016fast} and to a collaboration with SciLifeLab Stockholm, which resulted in the journal publication~\cite{engstrom2017transcriptomics}. During the summer 2015, I also visited the New York University and worked on a Markovian base caller for Oxford nanopore DNA sequencers (work presented at WCTA 2015 and at IBM New York, read section \emph{Workshops and Seminars}). 
%
%\paragraph{Programming}
%
%In addition to the bioinformatic software mentioned above, 
%I am the author of the first C++ library of compressed dynamic data structures for string processing \url{github.com/xxsds/DYNAMIC}\footnote{The library has been cited in \emph{Navarro, Gonzalo. Compact data structures: A practical approach. Cambridge University Press, 2016.}},  published in~\cite{prezzaFramework}, and of more than 13 C++ repositories (\url{github.com/nicolaprezza}).
%
%



\section*{Talks at international venues}

Below I report a list of the main talks I have given at international conferences and workshops. I will moreover present our work \emph{Dominik Kempa and Nicola Prezza. “At the Roots of Dictionary Compression: String Attractors.”} at the 50th Annual ACM Symposium on the Theory of Computing (STOC 2018). 

\subsection*{Conferences}

\begin{enumerate}
\setlength\itemsep{-1pt}
\item \emph{In-Place Sparse Suffix Sorting}. Twenty-Ninth Annual ACM-SIAM Symposium on Discrete Algorithms (SODA), January 7-10, 2018, New Orleans.
\item \emph{Optimal-time text indexing in BWT-runs bounded space}. Twenty-Ninth Annual ACM-SIAM Symposium on Discrete Algorithms (SODA), January 7-10, 2018, New Orleans.
\item \emph{Succinct Partial Sums and Fenwick Trees}. String Processing and Information Retrieval (SPIRE), September 26-29, 2017, Palermo.
\item \emph{From LZ77 to the Run-Length Burrows-Wheeler Transform, and Back}. Symposium on Combinatorial Pattern Matching (CPM), July 4-6, 2017, Warsaw.
\item \emph{A Framework of Dynamic Data Structures for String Processing}. Symposium on Experimental Algorithms (SEA), June 21-23, 2017, King's College London.
\item \emph{Space-Efficient Re-Pair Compression}.
Data Compression Conference (DCC), March 26-29, 2017, Snowbird (UT).
\item \emph{A Randomized LCE Data Structure} (best student talk award). Mathematical foundations for bioinformatics (MatBio), July 20, 2016, King's College London.
\item \emph{Computing LZ77 in Run-Compressed Space }.
 Data Compression Conference (DCC), March 29 - April 1, 2016, Snowbird (UT).
\item \emph{Fast Online Lempel-Ziv Factorization in Compressed Space}. String Processing and Information Retrieval (SPIRE), September 3, 2015, King's college London.
\item \emph{Average linear time and compressed space construction of the Burrows-Wheeler transform}. 9th International Conference on Language and Automata Theory and Applications (LATA), March 2-6, 2015, Nice, France.
\item \emph{Hashing and Indexing: succinct data structures and smoothed analysis}. Symposium on Algorithms and Computation (ISAAC), December 15-17, 2014, Jeonju, Korea.
\end{enumerate}


\subsection*{Workshops and Seminars}

\begin{enumerate}
	\setlength\itemsep{-1pt}
	\item \emph{String Attractors}. Workshop on Compression, Text and Algorithms (WCTA), September 29, 2017, Palermo, Italy.
	\item \emph{String Attractors}. NII Shonan Meeting: ``Computation over Compressed Structured Data'', October 9-12, 2017, Shonan Village center, Japan.
	\item \emph{In-place Longest Common Extensions}. Dagstuhl seminar: Computation over Compressed Structured Data, October 28, 2016, Schloss Dagstuhl - Leibniz Center for Informatics.
	\item \emph{Indexing in repetition-aware space}. Dagstuhl seminar: Computation over Compressed Structured Data, October 26, 2016, Schloss Dagstuhl - Leibniz Center for Informatics.
	\item \emph{Algorithms for the compression of genomic big data}. BITS annual meeting, June 15-17, 2016, University of Salerno.
	\item \emph{Compressed Indexes for Populations of Genomes}. EPIGEN annual meeting, May 24-27 2016, Hotel Ergife, Rome.
	\item \emph{Space-efficient compression and indexing of genomic big data: theory and practice}. Workshop on Data Structures in Bioinformatics (DSB), February 23-24, 2016, University of Bielefeld.
	\item \emph{Aligning Nanopore Events on a Reference Genome}. Workshop on Compression, Text and Algorithms (WCTA), September 4, 2015, King's college London.
	\item \emph{ONTRC: Bayesian Base-Calling of Nanopore Events for Portable Clinical Genomics}. IBM Thomas J. Watson Research Center, New York, July 30, 2015. Joint work with NYU's Courant Institute of Mathematcal Sciences (CIMS), and New York Genome Center (NYGC).
	\item \emph{Algorithms for the analysis of epigenetic data}. EPIGEN annual meeting, April 22, 2015, Hotel Ergife, Rome.
	\item \emph{Hashing and Indexing with succinct data structures}. Data Structures in Bioinformatics, December 8-9, 2014, Montpellier, France.
	\item{Hashing and Indexing: succinct data structures and smoothed analysis}. December 2, 2014, University of Helsinki, Department of computer science.
	\item \emph{Analysis of capture-based bisulfite sequencing data with ERNE-BS5}. NETTAB, October 16, 2014, University of Torino, molecular Biotechnology Center (MBC).
	\item \emph{Differential methylation analysis of target enrichment protocols data with ERNE-BS5}. EPIGEN NGS and DATA ANALYSIS WORKSHOP, April 11, 2014, Istituto di Genomica Applicata, Udine.
	\item \emph{A tour to DNA methylation analysis via BS-seq reads alignment}. March 26, 2014, SciLifeLab, Stockholm.
	\item \emph{Fast randomized approximate string matching with succinct hash data structures}. BITS Annual Meeting, February 26-28, 2014, Rome.
\end{enumerate}


\section*{Publications.}
Publications on computer science conferences and journals (i.e. \cite{Policriti2017} and all conferences except \cite{prezza2012erne}) follow the tradition of alphabetically sorted author lists.

\bibliographystyle{alpha}
%\bibliography{curriculum}
\nobibliography{curriculum}

\subsection*{Peer reviewed journal publications.}
\begin{tabular}{p{2cm} p{10cm}}
	\cite{Policriti2017} & \bibentry{Policriti2017}\\
	\cite{policriti2015fast1} & \bibentry{policriti2015fast1}\\
	\cite{prezza2016fast} & \bibentry{prezza2016fast}\\
	\cite{engstrom2017transcriptomics} & \bibentry{engstrom2017transcriptomics}\\
\end{tabular}

\subsection*{Peer reviewed conference publications.}

\noindent
\begin{tabular}{p{2cm} p{10cm}}
	\cite{kempa2018roots} & \bibentry{kempa2018roots}\\
	\cite{prezzaSparse} & \bibentry{prezzaSparse}\\
	\cite{prezzaOptimal} &
	\bibentry{prezzaOptimal}\\

\end{tabular}

\noindent
\begin{tabular}{p{2cm} p{10cm}}
	\cite{GNPlatin18} &
	\bibentry{GNPlatin18}\\
	\cite{billeFenwick} & \bibentry{billeFenwick}
\end{tabular}

\noindent
\begin{tabular}{p{2cm} p{10cm}}
	\cite{policriti2017lz77} & \bibentry{policriti2017lz77}\\
\end{tabular}
\begin{tabular}{p{2cm} p{10cm}}
	\cite{bille2017space} & \bibentry{bille2017space}\\
	\cite{belazzougui2017flexible} & \bibentry{belazzougui2017flexible}\\
\end{tabular}
\begin{tabular}{p{2cm} p{10cm}}
	\cite{prezzaFramework} & \bibentry{prezzaFramework}\\
	\cite{policriti2016computing} & \bibentry{policriti2016computing}\\
\end{tabular}
\begin{tabular}{p{2cm} p{10cm}}
	\cite{belazzougui2015composite} & \bibentry{belazzougui2015composite}\\
	\cite{policriti2015fast2} & \bibentry{policriti2015fast2}\\
\end{tabular}
\begin{tabular}{p{2cm} p{10cm}}
	\cite{policriti2015average} & \bibentry{policriti2015average}\\
	\cite{policriti2014hashing} & \bibentry{policriti2014hashing}\\
	\cite{prezza2012erne} & \bibentry{prezza2012erne}\\
\end{tabular}


%\vspace{30pt}

%Date\ \ \ \ November 26, 2017\hfill Signature



\end{document}